<ui version="4.0" >
 <class>PluginUninstallDialog</class>
 <widget class="QWidget" name="PluginUninstallDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Plugin Uninstallation</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Plugin directory:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="pluginDirectoryCombo" >
     <property name="toolTip" >
      <string>Select the plugin area containing the plugin to uninstall</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Plugin:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="pluginNameCombo" >
     <property name="toolTip" >
      <string>Select the plugin to uninstall</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pluginDirectoryCombo</tabstop>
  <tabstop>pluginNameCombo</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
